frontend/components/panels/DesktopSectionPanel.js

```diff
@@ -116,6 +135,22 @@ export default class DesktopSectionPanel extends React.Component {
 
     const honorsCheck = this.props.section.honors ? <Icon name='check' /> : <Icon name='x' />;
 
+    let notifBox;
+
+    if (this.props.section.seatsRemaining === 0) {
+      notifBox = (
+        <div data-tip='Sign up for notifications for this section' className='inlineBlock'>
+          <NotifCheckBox seats={ this.props.section.seatsRemaining } section={ this.props.section } />
```

Comment:
no need to pass both this.props.section.seatsRemaining and this.props.section, because you can get the seatsRemaining from this.props.section